### PR TITLE
Add telemetry properties to trial app commands

### PR DESCRIPTION
**Pull request by alexweininger**
Examples of queries we can make from these properties:
- How many trial app users are deploying their own project vs. the template?
- How much time do users take after importing a trial app to deploy (if ever)?
- How many users are logged in when they deploy their trial app?

### CODE REVIEW

File: src/commands/logstream/startStreamingLogs.ts [hunk 1]
```diff
@@ -25,6 +24,5 @@ export async function startStreamingLogs(context: IActionContext, node?: SiteTre
         }
     };
 
-    const client: ISimplifiedSiteClient = (node instanceof TrialAppTreeItem) ? node.client : node.root.client;
-    await appservice.startStreamingLogs(client, verifyLoggingEnabled, node.logStreamLabel);
+    await appservice.startStreamingLogs(node.client, verifyLoggingEnabled, node.logStreamLabel);
```
Comment (alexweininger) [inline src/commands/logstream/startStreamingLogs.ts:27]: This change is not related to telemetry, just removing the extra line of code.

File: src/commands/trialApp/importTrialApp.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export async function importTrialApp(_context: IActionContext, loginSession: str
             expirationDate: expirationDate,
             loginSession: loginSession
         };
-
+        context.telemetry.properties.trialTimeRemaining = String(trialAppNode.metadata.timeLeft);
```
Comment (nturinski) [inline src/commands/trialApp/importTrialApp.ts:33]: I feel like you might as well just use the addTrialAppTelemetry here too even though it's obviously going to be a trialApp and they _should_ not be logged in

Comment (nturinski) [reply src/commands/trialApp/importTrialApp.ts:33]: Ah but whatever, it doesn't matter.  Could just be cluttering up the properties too

Comment (nturinski) [reply src/commands/trialApp/importTrialApp.ts:33]: And there's no node, never mind lol sorry for all the comments
